app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from datetime import datetime
+from time import sleep
+
+
+def file_creating():
+    while True:
+        file_name = f"app-{datetime.today().strftime('%H_%M_%-S')}.log"
```

Comment:
please replace '%H_%M_%-S' with '%H_%M_%S' - otherwise the code doesn't work

Reply:
You are right. `-` before directive working only on Linux (I'm using Ubuntu), don't know about Mac - https://stackoverflow.com/questions/9525944/python-datetime-formatting-without-zero-padding.
It should just delete zeros in seconds.